.papr.sh

```diff
@@ -48,10 +48,10 @@ trap upload_journals ERR
 export ANSIBLE_LOG_PATH=ansible.log
 
 # run the prerequisites play
-ansible-playbook -v -i $PAPR_INVENTORY playbooks/prerequisites.yml
+ansible-playbook -vv -i $PAPR_INVENTORY playbooks/prerequisites.yml
```

Comment:
-vvv is what I usually use.  This gives all the input args to modules and other useful information.  I'm not sure offhand how it compares to -vv.

Reply:
Removed this commit, now its part of #8265